modules/ROOT/pages/queries-aggregations/filtering.adoc

```diff
@@ -178,34 +212,47 @@ Conversely, the following operator is available on array fields, and accepts a s
 
 These operators are available for all types apart from `Boolean`.
 
-== Logical operators
+== Filtering spatial types
 
-All operators can be combined using the logical operators `AND`, `OR`, and `NOT`.
-They can also be standalone operators, which means that they can be used as such and not be appended to field names.
+Both the `Point` and the `CartesianPoint` types use xref::queries-aggregations/filtering.adoc#_numerical_operators[numerical operators] and also have an additional `_DISTANCE` filter.
+Here is a list of what each filter does for the two types:
 
-These operators accept an array argument with items of the same format as the `where` argument, which means they can also be nested to form complex combinations.
+* `_LT`: checks if a point is less than the distance in the `distance` field away (in meters) from the point specified by the `point` field.
+* `_LTE`: checks if a point is less than or equal to the distance in the `distance` field away (in meters) from the point specified by the `point` field.
+* `_DISTANCE`: checks if a point is the exact distance in the `distance` field away (in meters) from the point specified by the `point` field.
+* `_GT`: checks if a point is greater than the distance in the `distance` field away (in meters) from the point specified by the `point` field.
+* `_GTE`: checks if a point is greater than or equal to the distance in the `distance` field away (in meters) from the point specified by the `point` field.
 
-For example, if you want to match all actors by the name of either "Keanu" or not belonging to the "Pantoliano" family, that played in "The Matrix" movie, here is how you can query that:
+For a `Point` type, all filters take the following type as an argument:
 
 [source, graphql, indent=0]
 ----
-query {
-    actors(where: { 
-        AND: [
-            { 
-                OR: [
-                    { name_CONTAINS: "Keanu" },
-                    { NOT: { name_ENDS_WITH: "Pantoliano" } }
-                ]
-            },
-            {
-                movies_SOME: { title: "The Matrix" }
-            }
-        ]}
-    ) {
+input PointDistance {
+    point: Point!
+    distance: Float!
+}
+----
+
+Similarly, for a `CartesianPoint` type, all filters take the following type as an argument:
+
+[source, graphql, indent=0]
+----
+input CartesianPointDistance {
+    point: CartesianPoint!
+    distance: Float!
+}
+----
+
+In practice, you can construct queries like the following which finds all users within a 5km (5000m) radius of a `Point`:
```

Comment:
this is what i mean - would this work with a `CartesianPoint` as well? then we could use this example for both

Reply:
It will look similar, the difference is that instead of `longitude` and `latitude` you will have `x` and `y`. 
For instance:
```
query CloseByUsers($x: Float!, $y: Float!) {
  users(where: { location_LTE: { point: { x: $x, y: $y }, distance: 5000 } }) {
    name
    location {
      x
      y
    }
  }
}
```

Reply:
right! makes sense. i'll add that